src/menu.c

```diff
@@ -6622,8 +6619,8 @@ s32 tt_menu_loop(void) {
                     case 3:
                         func_80001D04(0xEF, 0);
                         func_80036BCC(0x22E, 1);
-                        func_8009C674(&D_800E1E2C);
-                        allocate_menu_images(&D_800E1E40);
+                        func_8009C674(D_800E1E2C);
+                        allocate_menu_images(D_800E1E40);
```

Comment:
Looks like these should still be dereferenced?
```
/* 09EA48 8009DE48 3C04800E */  lui   $a0, %hi(D_800E1E2C) # $a0, 0x800e
/* 09EA4C 8009DE4C 0C02719D */  jal   func_8009C674
/* 09EA50 8009DE50 24841E2C */   addiu $a0, %lo(D_800E1E2C) # addiu $a0, $a0, 0x1e2c
/* 09EA54 8009DE54 3C04800E */  lui   $a0, %hi(D_800E1E40) # $a0, 0x800e
/* 09EA58 8009DE58 0C027229 */  jal   allocate_menu_images
/* 09EA5C 8009DE5C 24841E40 */   addiu $a0, %lo(D_800E1E40) # addiu $a0, $a0, 0x1e40
```

Reply:
I was just fixing this warning from GCC. Is there another way to do that?

![image](https://user-images.githubusercontent.com/828639/152422082-50763421-f082-4a80-937a-c5f017cfb60b.png)

Reply:
Ah, I didn't realize D_800E1E40 was an array. `allocate_menu_images(D_800E1E40)` and `allocate_menu_images(&D_800E1E40)` is identical behavior in this case, so removing the & is fine.